indra_db/managers/dump_manager.py

```diff
@@ -458,6 +450,13 @@ def dump(principal_db, readonly_db, delete_existing=False, allow_continue=True,
         else:
             logger.info("Statement dump exists, skipping.")
 
+        if not allow_continue or not Sif.from_list(starter.manifest):
+            logger.info("Dumping sif from the readonly schema on principal.")
+            Sif(db=principal_db, date_stamp=starter.date_stamp)\
```

Comment:
Is the db=principal_db actually required as an arg since the Sif dumper class doesn't use the DB anymore?